Pharmacy/Pharmacy/Model/Medicine.cs

```diff
@@ -1,8 +1,22 @@
-﻿namespace Pharmacy.Model
+﻿using System.Collections.Generic;
+
+namespace Pharmacy.Model
 {
     public class Medicine
     {
         public int Id { get; set; }
         public string Name { get; set; }
+        public int ManufacturerId { get; set; }
```

Comment:
Would change those List<T> to IEnumerable<T> (IEnumerable has extension .toList() if necessary)